sync/handlers.py

```diff
@@ -193,13 +193,14 @@ def __call__(self, git_gecko, git_wpt, body):
         logger.info("Setting taskgroup id for try push %r to %s" % (try_push, task_id))
         try_push.taskgroup_id = task_id
 
-        if result != "success":
+        if result != "success" or len(try_push.failed_builds):
             try_push.status = "complete"
             try_push.infra_fail = True
             sync = try_push.sync(git_gecko, git_wpt)
             logger.error("Decision task got status %s for task %s%s" %
```

Comment:
This logging isn't true, right? It's not the status of the decision task.

Reply:
TaskHandler runs when we get a taskgroup-resolved message about the decision task, so `body["result"]` is just the result of the decision task. But I will add an extra component to the message about build failures.